packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -386,7 +391,10 @@ test.serial('bulk operation BULK_GRANULE applies workflow to list of granule IDs
       granulePgRecord,
       knexOrTransaction: t.context.knex,
     });
-    t.deepEqual(matchingGranule, callArgs[0].granule);
+
+    const omitList = ['dataType', 'version'];
```

Comment:
Is this use of the `omitList` necessary? I pulled down your code and ran it and I didn't see these properties in the `matchingGranule`. Same question applies throughout the PR

Reply:
Ah good point, I'll update it

Reply:
[`e7c8575` (#2651)](https://github.com/nasa/cumulus/pull/2651/commits/e7c85758457ac5b23a91f74243623fec4bd7f640#diff-34fdd0872fdf91f43665d463ad1599f10c1e86debead4e375f82830473338724L395-L397)

Reply:
I removed it where you mentioned and also throughout the PR except for one test case where omitting those fields are necessary